src/web/modules/custom/work_bc_quiz/work_bc_quiz.post_update.php

```diff
@@ -115,6 +115,8 @@ function work_bc_quiz_post_update_350_2_noc_migration(&$sandbox = NULL) {
 
   $langcode = \Drupal::languageManager()->getCurrentLanguage()->getId();
 
+  $workbc_url = getenv('WORKBC_URL');
```

Comment:
in order to make absolutely sure a trailing slash won't mess us up, I like to do: `rtrim($str, '/') .'/'`

Reply:
code has been updated and migration re-tested